ray-operator/controllers/ray/common/service.go

```diff
@@ -1,12 +1,50 @@
 package common
 
 import (
-	rayiov1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
-	"github.com/ray-project/kuberay/ray-operator/controllers/ray/utils"
+	"context"
+	"fmt"
+
+	fmtErrors "github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	rayiov1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	"github.com/ray-project/kuberay/ray-operator/controllers/ray/utils"
 )
 
+func FetchDashboardURL(ctx context.Context, cli client.Client, rayCluster *rayiov1alpha1.RayCluster) (string, error) {
```

Comment:
Please check the `ray-operator/controllers/ray/rayservice_controller.go:442`. Do you need dashboard url or dashboard agent url? I think job submit still the `dashboard url`